src/sass/_shame.scss

```diff
@@ -138,3 +138,54 @@ body .row {
 .csp-inline-patch-logout {
   padding: 2em 0;
 }
+
+// /education/apply/ uses this expander button for navigation
+// Where else can I put this? Also, it could use a better name.
```

Comment:
Not a bad idea. How can I get that to be loaded by this file specifically? Just make another case in the parent layout, `page-breadcrumbs.html` to include an `include` (from `content/includes` which has a `<link>` to the css? (And actually, I guess that might make it the other way around; it'll do no-react.css by default and include this other little css file.)

The other consideration is the build process. Do we need to build another file and include it in only this? Seems a bit...excessive to me.

At any rate, this is only a temporary solution until we get multiform up and running, so maybe we do just leave it here in `_shame.scss` and if this is used elsewhere in the future, it can be addressed then?